packages/web-pkg/src/composables/piniaStores/modals.ts

```diff
@@ -47,7 +47,7 @@ export type Modal = {
   customComponent?: CustomModalComponent
   customComponentAttrs?: () => Record<string, unknown>
   onCancel?: () => void
-  onConfirm?: (value: string) => void | Promise<void>
+  onConfirm?: (value: any) => void | Promise<void>
```

Comment:
A bit more appropriate:

```suggestion
  onConfirm?: (value: unknown) => void | Promise<void>
```